docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -156,3 +156,14 @@ Sometimes you'll want to reset the data in your local environment. One common ca
    docker-compose --env-file .env.dev -f docker-compose.yaml -f docker-compose.dev.yaml up -V
   ```
 
+## Troubleshooting
+
+### `gradlew Could not target platform: 'Java SE 14' using tool chain: 'JDK 8 (1.8)'.`
+
+Somehow gradle didn't pick up the right java version for some reason.
+Find the install version and set the `JAVA_HOME` environment to point to the JDK folder.
+
+For example:
+```
+env JAVA_HOME=/usr/lib/jvm/java-15-openjdk ./gradlew  :airbyte-integrations:connectors:your-connector-dir:build
```

Comment:
```suggestion
env JAVA_HOME=/usr/lib/jvm/java-14-openjdk ./gradlew  :airbyte-integrations:connectors:your-connector-dir:build
```

Reply:
Good call. I think I have java 15 on my machine and that's where I needed this command. It seems to work fine with 15.